starboard/rdk/test_filters.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Starboard RDK Platform Test Filters."""
+
+from starboard.tools.testing import test_filter
+
+# pylint: disable=line-too-long
+_FILTERED_TESTS = {
```

Comment:
Can you add TODOs for any bugs that are tracking the work to fix this?

Reply:
b/302394149